messages/messages.go

```diff
@@ -6,18 +6,19 @@ var (
 	ERROR_INVALID_AIRFLOW_VERSION = "Unsupported Airflow Version specified. Please choose from: %s \n"
 	ERROR_NEW_MAJOR_VERSION       = "There is an update for Astro CLI. You're using version %s, but %s is the server version.\nPlease upgrade to the matching version before continuing. See https://www.astronomer.io/docs/cli-quickstart for more information.\nTo skip this check use the --skip-version-check flag.\n"
 
-	CLI_CMD_DEPRECATE         = "Deprecated in favor of %s\n"
-	CLI_CURR_VERSION          = "Astro CLI Version: %s"
-	CLI_CURR_COMMIT           = "Git Commit: %s"
-	CLI_CURR_VERSION_DATE     = CLI_CURR_VERSION + " (%s)"
-	CLI_LATEST_VERSION        = "Astro CLI Latest: %s"
-	CLI_LATEST_VERSION_DATE   = CLI_LATEST_VERSION + " (%s)"
-	CLI_INSTALL_CMD           = "\t$ curl -sL https://install.astronomer.io | sudo bash \nOR for homebrew users:\n\t$ brew install astronomer/tap/astro"
-	CLI_RUNNING_LATEST        = "You are running the latest version."
-	CLI_CHOOSE_WORKSPACE      = "Please choose a workspace:"
-	CLI_SET_WORKSPACE_EXAMPLE = "\nNo default workspace detected, you can list workspaces with \n\tastro workspace list\nand set your default workspace with \n\tastro workspace switch [WORKSPACEID]\n\n"
-	CLI_UPGRADE_PROMPT        = "A newer version of the Astronomer CLI is available.\nTo upgrade to latest, run:"
-	CLI_UNTAGGED_PROMPT       = "Your current Astronomer CLI is not tagged.\nThis is likely the result of building from source. You can install the latest tagged release with the following command"
+	CLI_CMD_DEPRECATE                 = "Deprecated in favor of %s\n"
+	CLI_CURR_VERSION                  = "Astro CLI Version: %s"
+	CLI_CURR_COMMIT                   = "Git Commit: %s"
+	CLI_CURR_VERSION_DATE             = CLI_CURR_VERSION + " (%s)"
+	CLI_LATEST_VERSION                = "Astro CLI Latest: %s"
+	CLI_LATEST_VERSION_DATE           = CLI_LATEST_VERSION + " (%s)"
+	CLI_INSTALL_CMD                   = "\t$ curl -sL https://install.astronomer.io | sudo bash \nOR for homebrew users:\n\t$ brew install astronomer/tap/astro"
+	CLI_RUNNING_LATEST                = "You are running the latest version."
+	CLI_CHOOSE_WORKSPACE              = "Please choose a workspace:"
+	CLI_SET_WORKSPACE_EXAMPLE         = "\nNo default workspace detected, you can list workspaces with \n\tastro workspace list\nand set your default workspace with \n\tastro workspace switch [WORKSPACEID]\n\n"
+	CLI_UPGRADE_PROMPT                = "A newer version of the Astronomer CLI is available.\nTo upgrade to latest, run:"
+	CLI_UNTAGGED_PROMPT               = "Your current Astronomer CLI is not tagged.\nThis is likely the result of building from source. You can install the latest tagged release with the following command"
+	CLI_DEPLOYMENT_HARD_DELETE_PROMPT = "\nThis operation is irreversible and permanent. Are you sure?"
```

Comment:
@paolaperaza, just wanted to confirm this sounds fine?

Reply:
Jake witz provided the following guidance for the UI. We should try to stay consistent here too: 

```
Hover text: "Deletes all infrastructure and records for this Deployment"

Warning popup: "Warning: This action permanently deletes all data associated with this Deployment. You will not be able to recover any part of your Deployment even with the assistance of Astronomer support. Proceed with the hard delete?"```

Reply:
@rdodev Agreed, no reason this warning should be different than what we should in the UI. I might suggest:

> Warning: This action permanently deletes all data associated with this Deployment, including the database. You will not be able to recover it. Proceed with hard delete?"